src/lando/static_src/legacy/js/components/Timeline.js

```diff
@@ -1,50 +1,51 @@
+        button.addClass("is-loading");
+        fetch(`/landing_jobs/${landing_job_id}`, {
+            method: 'PUT',
+            body: JSON.stringify({"status": "CANCELLED"}),
+            headers: {
+                'Accept': 'application/json',
+                'Content-Type': 'application/json',
+                'X-CSRFToken': button.data("csrf-token")
+            },
+        }).then(response => {
+            if (response.status == 200) {
+                window.location.reload();
+            } else if (response.status == 400) {
+                button.prop("disabled", true);
+                button.removeClass("is-danger").removeClass("is-loading").addClass("is-warning");
+                button.html("Could not cancel landing request");
+            } else {
+                button.prop("disabled", true);
+                button.removeClass("is-danger").removeClass("is-loading").addClass("is-warning");
+                button.html("An unknown error occurred");
+            }
+        });
+    });
 
-    button.addClass("is-loading");
-    fetch(`/landing_jobs/${landing_job_id}`, {
-        method: 'PUT',
-        body: JSON.stringify({"status": "CANCELLED"}),
-        headers: {
-            'Accept': 'application/json',
-            'Content-Type': 'application/json'
-        },
-    }).then(response => {
-        if (response.status == 200) {
-            window.location.reload();
-        } else if (response.status == 400) {
-            button.prop("disabled", true);
-            button.removeClass("is-danger").removeClass("is-loading").addClass("is-warning");
-            button.html("Could not cancel landing request");
-        } else {
-            button.prop("disabled", true);
-            button.removeClass("is-danger").removeClass("is-loading").addClass("is-warning");
-            button.html("An unknown error occurred");
-        }
+    $("a.toggle-content,button.toggle-content").on("click", function() {
+        /* A link with the `toggle-snippet` class will hide its parent, and show
+         * any of the parent's siblings. For example:
+         * <div>
+         *  <div>Some content <a href="#" class="toggle-content">toggle</a></div>
+         *  <div>Other content <a href="#" class="toggle-content">toggle</a></div>
+         * </div>
+         */
+        var link = $(this);
+        link.parent().hide();
+        link.parent().siblings().show()
     });
-});
 
-$("a.toggle-content,button.toggle-content").on("click", function() {
-    /* A link with the `toggle-snippet` class will hide its parent, and show
-     * any of the parent's siblings. For example:
-     * <div>
-     *  <div>Some content <a href="#" class="toggle-content">toggle</a></div>
-     *  <div>Other content <a href="#" class="toggle-content">toggle</a></div>
-     * </div>
-    */
-    var link = $(this);
-    link.parent().hide();
-    link.parent().siblings().show()
-});
+    return this.each(function() {
```

Comment:
I just noticed that in other JavaScript files we put all the code after this line, but this was moved to be outside of this block. Might have been a relic of the legacy implementation of these methods, but probably wouldn't hurt to continue following the general conventions as they are elsewhere.

Reply:
Yeah, you're right. This could have been a bit racy otherwise, but the way the main.js works is that it load and executes all the code when the document is ready https://github.com/shtrom/lando/blob/d94b2fcea886ce9387a30d8f68009902533d477a/src/lando/static_src/legacy/js/main.js#L9-L17, so it didn't cause a noticeable issue.